audio/freac/Portfile

```diff
@@ -25,7 +25,7 @@ github.tarball_from releases
 depends_lib         port:BoCA \
                     port:smooth
 
-depends_run         path:bin/ffmpeg:ffmpeg \
+depends_run         path:bin/ffmpeg:ffmpeg8 \
```

Comment:
This is not likely to work, since `ffmpeg8` should not install `ffmpeg` binary into `/opt/local/bin` (otherwise it cannot be co-installed with `ffmpeg` port). If the port requires `ffmpeg8`, then it should depend on `ffmpeg8`, and appropriate path is to be used.

Reply:
Right. In my local setup it was only working by accident, as I have ffmpeg installed. I now patched freac to use the ffmpeg8 binary.